src/F2_autoreport.py

```diff
@@ -219,6 +219,25 @@ def insert_image_array(img_list, width=500):
     )
 
 
+def insert_log_messages(log_dict):
+    """
+    :param log_dict: dict, containing the logging messages
+    :return: html.Div() element
+    """
+    return html.Div(
+        children=[
+            html.Ul(
+                children=[
+                    html.Li(
+                        className="grid-x small-10 list-log", children=[html.Span(v)]
```

Comment:
Yesterday I meant to use grid-x *instead* of Ul and Li not a mix of both

Btw, the parent div should have the classname `grid-x` and the child should have the classname `cell small-10`

Reply:
```
            html.Ul(
                children=[
                    html.Li(
                        className="list-log", children=[html.Span(k), v]
                    )
                    for k, v in log_dict.items()
                ]
            )
```

```
            # this div is equivalent to html.Ul 
            html.Div(
                className="grid-x",
                children=[
                    #This div is equivalent to html.Li
                    html.Div(
                        className="cell grid-x", 
                        children=[
                            #with html.Li you have automatic markers, here not
                            html.Div(children=k, className="cell small-1 list-marker"), 
                            # this is what is inside the Li in the above example
                            html.Div(children=v, className="cell small-11 list-log")
                         ]
                    )
                    for k, v in log_dict.items()
                ]
            )
```

Reply:
If you make yourself drawings it becomes clearer

Reply:
I used the bullet point symbol directly when creating the dictionary for the error messages. It works fine without the second div. 

I removed the U1 and Li elements as you suggested in the latest commit for the screen and only retained them in the div for the print version.